spec/features/event_spec.rb

```diff
@@ -1,6 +1,15 @@
 require "rails_helper"
 
 describe "Event", type: :feature do
+  describe "index page" do
+    it "should link to the show page when an event title is clicked" do
+      event = FactoryGirl.create :event
+      visit events_path
+      click_link "Event-Name"
```

Comment:
I'd rather write `event.name` so that nothing breaks if someone changes the factory